content/kubermatic/master/tutorials_howtos/external_clusters/eks/_index.en.md

```diff
@@ -5,7 +5,54 @@ weight = 7
 
 +++
 
+## Add EKS Cluster
 
+You can add an existing Kubernetes cluster and then manage it using KKP.
+From the Clusters page, click `External Clusters`. Click `Add External Cluster` button and Pick `Elastic Kubernetes Engine` provider.
```

Comment:
s/Clusters/clusters
s/Pick/pick

Check also the rest of the docs for similar issues.